packages/color-helpers/types/global.d.ts

```diff
@@ -1,5 +0,0 @@
-export {};
-
-declare global {
-	type Color = [number, number, number];
-}
```

Comment:
This worked, but it wasn't technically correct.
`Color` is not a globally available type in node or any other runtime.

Using `import type` all over the place is messier but more accurate.

Reply:
I thought this was meant for these cases but good to know it isn't!